### ISSUE TITLE
Calls REST API to provide number of concurrent calls per account/client

### ISSUE THREAD

**Issue by gvagenas** [RestComm/Restcomm-Connect#1589]

**Comment by forever3000** [comment]
Hi @gvagenas ,
Is this total of concurrent calls per account/client?

**Comment by deruelle** [comment]
@forever3000 yes we need current # of concurrent calls per account and per client. We also need to the UsageAPI to report the max # of concurrent calls that was done in a Day, Week, Month, AllTime, ...
